apps/cpp_rpc/rpc_env.cc

```diff
@@ -164,8 +163,11 @@ void LinuxShared(const std::string output,
      cmd += " " + *f;
     }
     cmd += " " + options;
-    std::string executed_result = common::Execute(cmd);
-    CHECK(executed_result.length() == 0) << executed_result;
+    std::string err_msg;
+    auto executed_status = common::Execute(cmd, err_msg);
+    if (executed_status) {
+      LOG(ERROR) << err_msg;
```

Comment:
Use ```LOG(FATAL) << "RuntimeError: " << err_msg``` . This will allow us to throw a specific RuntimeError in the python side, by default ERROR may not throw an exception.

Reply:
done